### ISSUE TITLE
iOS - Chat - After sending an image, first double tap causes the image preview to disappear

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#31813]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.3-2
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause-Internal Team
**Slack conversation:** @

## Action Performed:
Pre-requisite: user must be logged in.

1. Go to any chat.
2. Send an image.
3. Tap on the image to see the preview.
4. Double tap to zoom in.

## Expected Result:
The image preview should be zoomed in.

## Actual Result:
The image preview disappears.

Also, as a similar behavior (double tap issue), when you open an image preview you already sent and double tap on it, sometimes the image does not zoom in, it just display or stops displaying the arrow buttons.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [x] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/4f40ccd4-f043-4243-9fc6-60631e98102a

https://github.com/Expensify/App/assets/115492554/69b487d3-bd48-45f3-9900-eee77414f715

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by paultsimura** [comment]
It's worth holding until https://github.com/Expensify/App/pull/31308 is merged.

**Comment by kadiealexander** [comment]
Yeah I can't reproduce this any more, closing as completed via https://github.com/Expensify/App/pull/31308
